apps/web/pages/api/teams/[team]/invite.ts

```diff
@@ -90,7 +90,7 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
         from: session.user.name,
         to: usernameOrEmail,
         teamName: team.name,
-        joinLink: `${BASE_URL}/auth/signup?token=${token}&callbackUrl=${BASE_URL + "/settings/teams"}`,
+        joinLink: `${BASE_URL}/auth/signup?token=${token}&callbackUrl=/settings/teams}`,
```

Comment:
To further avoid it altogether, we should have a method to generate a full link so that it can validate that absolute URL is wrong and relative URL is required.

```
getSignupLink({callbackUrl:`/settings/teams`, token})
```